### DISCUSSION TITLE
Installer Error - Can someone help with this error message? Have tried to reinstall the installer but no luck

### DISCUSSION THREAD

**Discussion by Annolago** [flybywiresim/aircraft#5022] [Help] [answered]
Hi
So a couple of days, maybe a week ago my installer went haywire and displayed the attached image when ever I started it to do an update. Yes, I did try to reinstall it but no luck. So as written in the error message I am now seeking help here..
Forgive me for erasing my name in the error message, but keen on privacy..
![FBW installer](https://user-images.githubusercontent.com/75162215/120912200-948eb180-c68d-11eb-85f2-5c9642ca2e22.jpg)
Hope someone can help

**Comment by Annolago** [comment, marked as answer]
Well turned out to be my MS Flight sim installment that for some odd reason went belly up. Had to delete it and are now currently reinstalling it.. Thank god for gigabit speed internet connection.

**Comment by Annolago** [comment]
So, 176 GB later, then another partly install of 80 some gigs everything is working perfectly.
Turns out that if your Fligtsim install is damaged you get locked out of the community folder, hence the errors from the installer. 
Hope this can help others.

**Reply by ExampleWasTaken** [reply]
A quick look at the [docs](https://docs.flybywiresim.com/start/reported-issues/#installer-issues) or asking in the [Discord support](https://discord.gg/flybywire) channel would have saved you from reinstalling. ;)

**Comment by Annolago** [comment]
No, could not even start Flightsim. So error was that the installer could not find the installer directories as they had been locked by the copy protection/rights management. Hence the reinstall, but thanks for the link anyway...
